eZ Platform/Admin UI/USE_CASE.md

```diff
@@ -0,0 +1,25 @@
+# Work in Progress
+
+# Prototype: Hybrid Application
+## Goals:
+- Validate hypothesis of a hybrid application architecture.
+- Learn feasibility and difficulties
+
+## Stories
+
+### As a developer I want to simply reuse Admin Panel functionalities present in 1.x
+- ex: content type view
+
+### As a developer I want to simply reuse existing YUI-based modules present in 1.x
+- ex1: universal discovery widget
+- ex2: landing page editor
+- ex3: sub-item views in the repository
+
+### As a developer I want to extend the hybrid app with a React-based module.
```

Comment:
I agree - the server side render should be the first step (or phase, depending on how involved it becomes). That's unique from the goal of this prototype, though, which has a goal of demonstrating the notion of "hybrid": server side rendering cross-bred with enhanced JS.

I think we wrote down React here as one of many desirable experiments, under the assumption that it was the defacto preferred first step to play with. I am completely agnostic on which library or framework is chosen for this experiment, including none at all.

The closer this prototype is to your vision of how-things-could-be, the more we can learn from it, but it does not have to tackle all of these goals at once if that adds complexity to the experiment.